fhirpathpy/__init__.py

```diff
@@ -35,11 +35,31 @@ def apply_parsed_path(resource, parsedPath, context=None, model=None, options=No
             (options or {}).get("userInvocationTable", {})
         ),
     }
+    
+    # Add trace callback if provided in options
+    if options and "trace_callback" in options:
+        ctx["trace_callback"] = options["trace_callback"]
```

Comment:
We use camelCase for options/ctx, and I suggest using `traceFn` for similariry with fhirpath.js